### PR TITLE
fix(deps): bump Swiper from 4.5.1/6.5.1 to 8.3.1

### PR DESCRIPTION
**Pull request by julien-deramond**
Closes https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1313

Indeed the dependency has been updated in the `package*.json` files without modifying the examples, the CSS, the CDNs, etc.

- [x] Bump Swiper to the most recent version: 8.3.1 in `package.json`, `assets/js/vendor`, CDNs, etc.
- [x] Adapt the examples to add `.swiper`
- [x] Adapt the CSS to position correctly prev and next buttons
- [x] Added a warning callout to explain that IE support was dropped in Swiper v5 

### Live previews
- [Components > Custom Carousel](https://deploy-preview-1314--boosted.netlify.app/docs/4.6/components/custom-carousel)
- [Examples > Orange Homepage](https://deploy-preview-1314--boosted.netlify.app/docs/4.6/examples/orange-homepage/)
- [Bootswatch](https://deploy-preview-1314--boosted.netlify.app/docs/4.6/boostwatch/#custom-carousel)

### CODE REVIEW

**Comment by louismaximepiton** [comment]
As far as I could test myself, we can bump to `v8.2.4` without any breaking change, multiple functions were added.

**Comment by julien-deramond** [comment]
There is a regression with the 6.5.1 on desktop when you swipe, it doesn't stop with a slide completely displayed but rather when you stop the movement (see screenshot). Same behavior observed with 6.5.1 and 8.2.4.

**Comment by louismaximepiton** [comment]
For what I can see, if we need to go on 6.5.1 or after, there's no change between 6.5.1 and 8.2.4. However, the breaking change appear at the 6.0.0 and I can't figure out what causes the breaking change in the [release note](https://swiperjs.com/changelog#6-0-0-released-on-july-3rd-2020) do you have any idea ?

**Comment by julien-deramond** [comment]
> For what I can see, if we need to go on 6.5.1 or after, there's no change between 6.5.1 and 8.2.4.

Yes agree on that. In this PR, let's go directly to 8.2.4.
So still need to check why there's a regression. Can't figure it out atm neither.

**Comment by julien-deramond** [comment]
No regression with the 8.2.6 so seems good now :)

File: package.json [hunk 1]
```diff
@@ -141,7 +141,7 @@
     "sinon": "^7.5.0",
     "stylelint": "^14.8.5",
     "stylelint-config-twbs-bootstrap": "^3.1.0",
-    "swiper": "^6.5.1",
+    "swiper": "^8.2.6",
```
Comment (louismaximepiton) [inline package.json:144]: Need to change the version l.181 too ?

File: package.json [hunk 2]
```diff
@@ -92,7 +92,7 @@
   "peerDependencies": {
     "jquery": "1.9.1 - 3",
     "popper.js": "^1.16.1",
-    "swiper": "^6.5.1",
+    "swiper": "^8.2.6",
```
Comment (louismaximepiton) [inline package.json:95]: Do we need to import swiper since we import it via cdn each time we use it in the doc ?

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
I have two questions more about the changes: 
- [x] Need to update `orange-homepage` example too ?
- [x] Need to update `site\static\docs\4.6\assets\js\vendor\swiper.min.js` too ?

**Comment by julien-deramond** [comment]
> Need to update orange-homepage example too ?

Yes indeed

> Need to update site\static\docs\4.6\assets\js\vendor\swiper.min.js too ?

I completely deleted it and used the CDN version instead in `site/layouts/partials/scripts.html`. Should still work like that.

**Comment by julien-deramond** [comment]
Need to work on this since when I change the version everywhere, the CSS is broken

**Comment by julien-deramond** [comment]
Bumped the version to 8.3.0 (releases 7 days ago) and modified the CSS to fix the broken rendering of prev/next buttons.

**Comment by julien-deramond** [comment]
:warning: Just saw that IE and old browser support were dropped in v5.0.0. Maybe we should stay in v4.5.1... and simply revert what's been updated in the `package*.json` (bump to v5.6.1).

File: config.yml [hunk 3]
```diff
@@ -93,10 +93,10 @@ params:
     js_bundle_hash:                 "sha384-2jsvnZnyYAg6xCGTSIdBPoEt3xP64FauXZlBK9PAYsneWVkqXT/5keQRlbdfXkCL"
     jquery:                         "https://code.jquery.com/jquery-3.5.1.slim.min.js"
     jquery_hash:                    "sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
-    css_swiper:                     "https://cdn.jsdelivr.net/npm/swiper@4.5.1/dist/css/swiper.min.css"
-    css_swiper_hash:                "sha384-3ycIs75n6btZktoCV/zcWo54Eq+bnjaJ9iwOISgNPQIXRSmrkf4XP1iHsWNq0jPB"
-    js_swiper:                      "https://cdn.jsdelivr.net/npm/swiper@4.5.1/dist/js/swiper.min.js"
-    js_swiper_hash:                 "sha384-llVNZVxgabZyf5ZeGs3m2QcNbEE0UK1PBKM6ZoJmWK5BuBpqZUXpN1nWXd0SrAC5"
+    css_swiper:                     "https://unpkg.com/swiper@8.3.0/swiper-bundle.min.css"
```
Comment (julien-deramond) [inline config.yml:96]: TODO:
* Switch to 8.3.1
* Revert unpkg.com and use https://cdn.jsdelivr.net/npm/swiper@8.3.1/swiper-bundle.min.css

File: scss/_o-carousel.scss [hunk 4]
```diff
@@ -47,10 +47,6 @@
   /* rtl:raw:
   transform: rotate(180deg);
   */
-
-  &::after {
```
Comment (julien-deramond) [inline scss/_o-carousel.scss]: @isabellechanclou Without those 2 `&::after` rules we can see blue arrows:

Screenshot from 2022-07-21 08-04-39

Comment (julien-deramond) [reply scss/_o-carousel.scss]: Reintroduced CSS via 886dcbba351b0ca81787112d148e6f57ece9264a

**Pull request merged**